reference-architecture/osp-cli/README.adoc

```diff
@@ -0,0 +1,152 @@
+== Scripted Installation of OpenShift on OpenStack.
+
+The scripts presented here match those in https://access.redhat.com/articles/2743631[Red Hat OpenShift Container Platform 3 on Red Hat OpenStack Platform 8] reference architecture document.  These script fragments are intended to demonstrate the steps needed to deploy OCP on OSP, but they are not the recommended method for doing so.  Production deployment should be done using the https://github.com/openshift/openshift-on-openstack[OpenShift on OpenStack] Heat templates and the https://github.com/openshift/openshift-ansible[OpenShift Ansible] playbooks.
+
+The documentation here does not provide comprehensive usage
+instructions for these script fragments.  The scripts are meant to be
+used in conjunction with the reference architecture document above.
+
+These scripts are offered as-is, and no warranty is implied or offered.
+
+=== Prerequisites
+
+To use these scripts require a DNS server running BIND.
+It must have a delegated sub-domain for the OpenShift service.  In
+these examples that subdomain is `osp3.example.com`. Adjust this for
+the target environment.  The subdomain must allow dynamic updates using
+`nsupdate`. The process requires a copy of the `rndc.key`
+from the BIND server to allow updates.
+
+The scripts require an _haproxy_ proxy server configured. The user
+must have an SSH key to allow access to the proxy server and the user
```

Comment:
"must have **an SSH key** to allow access"

I know what you mean, but something like "must have Key-Based SSH passwordless access"